content/en/openpnp/calibration/camera-fisheye-cal/index.md

```diff
@@ -7,28 +7,54 @@ description: >
   Applying fisheye calibration to the cameras
 ---
 
-Awesome! Now we're connected to the machine, our next step is calibrating the fisheye out of the camera lenses. You can reference the orignal OpenPnP documentation about this step [here](https://github.com/openpnp/openpnp/wiki/Camera-Lens-Calibration).
+Now we're connected to the LumenPnP, we need to set up the cameras so that they can accurately fine tune the machine's position. The LumenPnP is a robust machine, but it needs both cameras to make sure it can pick and place components reliably every time. Note that these are very important calibration steps, but OpenPnP makes them relatively easy. Certain steps will probably need you to tune them more than once to get them perfect, others should stay mostly constant.
```

Comment:
```suggestion
Now that you're connected to the LumenPnP, you'll need to set up the cameras so that they can accurately fine tune the machine's position. The LumenPnP is a robust machine, but it needs both cameras to make sure it can pick and place components reliably every time. Note that these are very important calibration steps, but OpenPnP makes them relatively easy. Certain steps will probably need you to tune them more than once to get them perfect, others should stay mostly constant.
```